tabular/tests/unittests/test_tabular.py

```diff
@@ -138,6 +139,36 @@ def test_advanced_functionality():
     assert len(leaderboard) == len(leaderboard_loaded)
     assert predictor_loaded.get_model_names_persisted() == []  # Assert that models were not still persisted after loading predictor
 
+    assert predictor.get_size_disk() > 0  # Assert that .get_size_disk() produces a >0 result and doesn't crash
+    # Test cloning logic
```

Comment:
Just a comment in general: we might want to split `test_advanced_functionality` into smaller functions. Currently it's quite long and kind of hard to read

Reply:
I moved the disk size check to a new function, however moving other things is somewhat tricky: We want to ensure that the combination of all the advanced things work as expected, so testing them individually isn't enough.